### ISSUE TITLE
php 7.3 incompatibility

### ISSUE THREAD

**Issue by dunkoh** [cirrusidentity/simplesamlphp-module-ratelimit#4]
Just updated my local environment to php 7.3 and error during logout/login when this module is referenced

`"continue" targeting switch is equivalent to "break". Did you mean to use "continue 2"?`

/vendor/simplesamlphp/simplesamlphp/modules/ratelimit/lib/Auth/Source/RateLimitUserPass.php:176

```
switch ($result) {
     case 'allow':
           Logger::debug('User \'' . $username . '\' login attempt allowed by ' . get_class($limiter));
           return true;
     case 'block':
           Logger::stats('User \'' . $username . '\' login attempt blocked by ' . get_class($limiter));
           return false;
      case 'continue':
           continue;
      default:
           Logger::warning("Unrecognized ratelimit allow() value '$result'");
}
```

**Comment by tvdijen** [comment]
It's a warning, not an error, but it should indeed be changed to `continue 2;` if the intended behaviour is to skip to the next iteration of the for-loop

**Comment by dunkoh** [comment]
I have a PR that will be ready shortly.

**Comment by dunkoh** [comment]
#5

**Comment by pradtke** [comment]
Thanks @dunkoh for the change, and @tvdijen for the review.  I've released v1.1.0 with this change.
